docs/sources/administration/view-server-stats.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "View server stats"
+type = "docs"
+keywords = ["grafana", "server", "statistics"]
+[menu.docs]
+weight = 1150
++++
+
+# View Grafana server stats
+
+If you are a Grafana server admin, then you can view useful statistics about your Grafana server in the Stats tab.
+
+> **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../permissions/_index.md" >}}).
+
+## View server stats
+
+1. Log in to your Grafana server with an account that has the Grafana Admin flag set.
+1. Hover your cursor over the **Server Admin** (shield) icon in the side menu and then click the **Stats** tab.
+
+## Available stats
+
+The following statistics are displayed in the Stats tab:
```

Comment:
It's worth mentioning here somehow that if users belong to several organisations then each user is counted once (for the highest role in any org) so as Total users = admins + editors + viewers. Same for active users.
For more details you can check the respective [PR](https://github.com/grafana/grafana/pull/26777) and [issue](https://github.com/grafana/grafana/issues/26744).

Reply:
I did not know that! Please take a look at my example and let me know if it makes sense.

For active users, do you need to log in under the particular organization in order to count as a user? For example, if you didn't log in to two of your four organizations, would you not count as an active user for those organizations?

Reply:
AFAIK active users are considered all the users that have signed in [the last month](https://github.com/grafana/grafana/blob/0bfdf79968e926aa436dd29ce17e582d06f93643/pkg/services/sqlstore/stats.go#L22). They are counted once regardless on how many organisations belong. Their role is the highest on all the organisations.

So for your example, assuming that Sofia is the only user then it should be:

- Total users     1
- Total admins    1 (the highest role of Sofia is Admin)
- Total editors   0
- Total viewers   0

@sakjur can you confirm that?

Reply:
Thanks, I clarified the text.

Reply:
Yes! And these stats are global for the instance, not per org.